lib/to_currency_select_tag.rb

```diff
@@ -10,6 +11,7 @@ module ToCurrencySelectTag
       def to_currency_select_tag(priority_currencies, options, html_options)
         html_options = html_options.stringify_keys
         add_default_name_and_id(html_options)
+        byebug
```

Comment:
This look like a left-over debug statement. Could you please remove it?

Reply:
oops. Fixed